var/spack/repos/builtin/packages/e3sm-scream/package.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class E3smScream(Package):
+    """Builds dependencies for E3SM SCREAM"""
+
+    homepage = "https://e3sm.org/"
+    url = "https://github.com/E3SM-Project/scream"
+    git = "https://github.com/E3SM-Project/scream.git"
+
+    # maintainers = ["Jessicat-H", "mtaylo12"]
+
+    version("master", branch="master", submodules=True)
+
+    depends_on("cmake@3.18.0")
+    depends_on("netcdf-fortran@4.4.4")
+    depends_on("netcdf-c@4.4.1")
```

Comment:
Do you have to be that precise all the way to  patch version? Can you use lower/upperbounds instead?